cloud/client.go

```diff
@@ -861,15 +1027,30 @@ func (c *client) WhoAmI() (string, string, bool, error) {
 	authType := "ssh"
 	if c.password != "" {
 		authType = "password"
-	} else if c.authToken != "" {
+	} else if c.authCredToken != "" {
 		authType = "token"
 	}
 
 	return pingResponse.Email, authType, pingResponse.WriteAccess, nil
 }
 
-func (c *client) getAuthTokenPath(create bool) (string, error) {
-	confDirPath := c.authTokenDir
+func (c *client) migrateV1Token() error {
+	confDirPath := c.authDir
+	if confDirPath == "" {
+		confDirPath = cliutil.GetEarthlyDir()
+	}
+	tokenPath := filepath.Join(confDirPath, "auth.token")
+	newPath := filepath.Join(confDirPath, "auth.credentials")
+	if fileutil.FileExists(tokenPath) {
```

Comment:
what if `newPath` exists too?

Reply:
under normal circumstances it shouldn't be an issue because `auth.token` is removed. 

A more complicated scenario: you upgrade to jwt, downgrade and changed your password, upgrade again back to jwt. The file renaming still succeeds, and ends up overwriting theold  `auth.credentials` with the new set of credentials containing the new password. Seems like that might actually be a good thing.

Any scenarios you can think of that could be an issue?

Reply:
Possibly down the road (e.g. in v1) we decide we need new data in `auth.credential`, then a user goes from earthly `v1` back to `v0.6.3`, then moves on to `v0.6.5` (the release which will contain this change), it'll then destroy data that was set in `v1`.

Reply:
I'm on the fence with this one. Seems like there's pros/cons either way. I'm perhaps more tempted to leave it as-is, since users downgrading back from JWT and changing their credentials or auth type seems more likely immediately than an incompatible change to how we store the `auth.credentials` file.